### PR TITLE
solution to problem 9 in js1

### PR DESCRIPTION
**Pull request by pxa4576**

### CODE REVIEW

File: curriculum/js1/9.js [hunk 1]
```diff
@@ -4,8 +4,12 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a=0) => {
+  if(a%7===0)
```
Comment (songz) [inline curriculum/js1/9.js]: solution(8) would return 21 right? did you run tests?

Comment (pxa4576) [reply curriculum/js1/9.js]: Yes I ran tests and all 4 of them passed

File: curriculum/js1/9.js [hunk 2]
```diff
@@ -4,8 +4,10 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a=0) => {
+  if((a+1)%7===0)
```
Comment (danielthalhuber) [inline curriculum/js1/9.js:8]: Please add a space after keywords like `if`, `for`, and `return`. In this case, you want `if ((a+1...`

File: curriculum/js1/9.js [hunk 3]
```diff
@@ -4,8 +4,10 @@
...
+    return a+1;
```
Comment (danielthalhuber) [inline curriculum/js1/9.js:9]: `if ((a+1)%7===0) return a+1;`

**Review by danielthalhuber — APPROVED** [review]

**Pull request merged**